README.md

```
@@ -12,6 +12,29 @@ FoodGram is a full-stack video-sharing web application, similar to TikTok or Ins
 * **Partner Profiles:** Users can visit the profile pages of the food partners.
 * **Video Upload:** Food partners can create and upload new food items/videos.
 
+---
+## 📸 Screenshots
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)